### PR TITLE
[react-article-components] Add load-more and update relateds styles

### PR DESCRIPTION
**Pull request by YuCJ**
KanbanFlow task: https://kanbanflow.com/t/zWWBLo1j

Design: https://kanbanflow.com/t/E46FLGmc
> 此版本主要調整以下三部分：
> ① 新增「載入更多」按鈕
> ② Hover「載入更多」按鈕的樣式
> ③ 正在載入時卡片的照片樣式

### CODE REVIEW

File: packages/react-article-components/src/components/related/list.js [hunk 1]
```diff
@@ -59,39 +172,58 @@ export default class List extends React.PureComponent {
...
     const { data } = this.props
-    const { isMounted } = this.state
-
     if (data.length === 0) {
       return null
     }
+    const { showAll } = this.state
+    const cards = showAll
+      ? _.map(data, this.renderCard)
+      : _.map(data.slice(0, firstShowedLimit.desktopAndAbove), this.renderCard)
```
Comment (nickhsine) [inline packages/react-article-components/src/components/related/list.js]: Problem:
We use `this.state.isMounted` to give each card `align-self: stretch` or `align-self: flex-start` style.
`stretch` to make sure card having the same height after mounting.
`flex-start` to make sure card having mouse hovering animation effects.

However, when the  time hidden cards `display: none` become `display: block`, `this.state.isMounted` is already `true`, which makes cards not have the same height.

The simple solution could be(for your reference)
```
// this.state.isMounted -> this.state.alignSelf
this.state = {
  showAll: false,
  alignSelf: 'stretch',
}

componentDidMount() {
  this.setState({
    alignSelf: 'flex-start',
  })
}

loadMore = () => {
  this.setState({
    showAll: true,
    alignSelf: 'stretch',
  }, () => {
    setTimeout(() => {
      this.setState({
        alignSelf: 'flex-start',
      })
    }, 500)
  })
}

renderCard = (item, i) => {
  ...
  <Item
     alignSelf={this.state.alignSelf}
     ...
  >
  ...
}
```

Comment (YuCJ) [reply packages/react-article-components/src/components/related/list.js]: Fixed

**Comment by YuCJ** [comment]
Updated

**Comment by nickhsine** [comment]
👍 👍  👍

**Pull request merged**